src/main/java/at/hannibal2/skyhanni/events/entity/EntityOpacityActiveEvent.kt

```diff
@@ -0,0 +1,18 @@
+package at.hannibal2.skyhanni.events.entity
+
+import at.hannibal2.skyhanni.api.event.SkyHanniEvent
+
+/**
+ * Fires once per second, to enable the [EntityOpacityEvent].
+ */
+class EntityOpacityActiveEvent : SkyHanniEvent() {
+    private var status = false
+
+    fun setActive(status: Boolean = true) {
```

Comment:
this shouldnt be allowed to set to false

Reply:
not sure what you mean. the idea is to pass a boolean to that function that might be false, in that case there should nothing happen. instead of that the caller site needs to wrap this function call in a if check